docs/platforms/javascript/common/enriching-events/process-isolation/index.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Process Isolation
```

Comment:
m: I'm not sure if "Process" isolation is a good term for the heading. We worry about isolating certain jobs/tasks/requests _within_ one process. 

I'd rather call this page "Request Isolation" or more generally "Task Isolation". WDYT?

Reply:
Yeah, I was also not 100% sure about this. My first instinct was also to go with `Request Isolation` but this does not really cover e.g. isolation background jobs 🤔 but maybe request isolation is still more understandable 🤔

Reply:
I vote for "Request Isolation". I think it's most clear and I think that it's ok that it doesn't cover everything. "Data isolation" is super broad and can mean just about anything. Same thing with "Process isolation".

Reply:
will rename it!